content/admin.py

```diff
@@ -64,6 +67,7 @@ class CustomPageAdmin(admin.ModelAdmin):
 admin.site.register(VolunteerPeru, VolunteerPeruAdmin)
 admin.site.register(VolunteerOpenPosition, VolunteerOpenPositionAdmin)
 admin.site.register(CustomPage, CustomPageAdmin)
+admin.site.register(Home, HomeAdmin)
```

Comment:
Since youre showing all the model fields, do you need to make a `HomeAdmin` class? 

I think you can do `admin.site.register(Home)` instead?